app/main.py

```
@@ -6,7 +6,7 @@
 import sys
 from typing import Dict, List, Optional, Tuple
 
-from fastapi import FastAPI, Request, HTTPException, status
+from fastapi import FastAPI, HTTPException, Request, status
```

Ruff (0.8.2)

`fastapi.HTTPException` imported but unused

Remove unused import: `fastapi.HTTPException`

(F401)